### DISCUSSION TITLE
复制封面原链接成功时应显示「✔」图标

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#4002] [功能建议 / Ideas]
即 https://github.com/the1812/Bilibili-Evolved/discussions/1304#discussioncomment-193633 提到的，在 [v1.11.12](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.12) 实现的细节。

V2 是忘做了还是只是目前失效了？

***

另外 Discussions 也有类似 Issue Forms 的 Category Forms 了：

- https://github.blog/changelog/2023-02-03-category-forms-on-github-discussions/
- https://github.blog/2023-01-09-github-discussions-just-got-better-with-category-forms/
- https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms

可以用起来引导填脚本版本和浏览器版本什么的（#1297）。

**Comment by the1812** [comment]
应该是没搬过来吧, 看原贴就是换 v2 那段时期的
